nullarbor/2.0.20191013/Dockerfile

```diff
@@ -0,0 +1,395 @@
+ rm -v v${CENTRIFUGE_VER}.tar.gz && \
+ cd centrifuge-${CENTRIFUGE_VER} && \
+ make && \
+ make install prefix=/usr/local && \
+ mkdir -v /centrifuge-db
+
+# iqtree
+RUN wget https://github.com/Cibiv/IQ-TREE/releases/download/v${IQTREE_VER}/iqtree-${IQTREE_VER}-Linux.tar.gz && \
+ tar -xzf iqtree-${IQTREE_VER}-Linux.tar.gz && \
+ rm iqtree-${IQTREE_VER}-Linux.tar.gz
+
+# FastTree binary download; as of 2022-06-01 downloads version 2.1.11
+RUN mkdir -v /FastTree && \
+ wget http://www.microbesonline.org/fasttree/FastTree -P /FastTree && \
+ chmod +x /FastTree/FastTree
+
+# quicktree
+RUN wget https://github.com/khowe/quicktree/archive/refs/tags/v${QUICKTREE_VER}.tar.gz && \
+ tar -xzf v${QUICKTREE_VER}.tar.gz && \
+ rm -v v${QUICKTREE_VER}.tar.gz && \
+ cd /quicktree-${QUICKTREE_VER} && \
+ make
+
+# newick_utils
+# no releases or tags, so manually setting to latest commit (da12115) to master branch
+RUN git clone https://github.com/tjunier/newick_utils.git && \
+ cd newick_utils && \
+ git reset --hard da12115 && \
+ autoreconf -fi && \
+ ./configure && \
+ make && \
+ make check	&& \
+ make install
+
+# DL Kraken2, unpack, and install
+RUN wget https://github.com/DerrickWood/kraken2/archive/v${KRAKEN2_VER}.tar.gz && \
+ tar -xzf v${KRAKEN2_VER}.tar.gz && \
+ rm -rf v${KRAKEN2_VER}.tar.gz && \
+ cd kraken2-${KRAKEN2_VER} && \
+ ./install_kraken2.sh . && \
+ mkdir -v /kraken2-db /kraken-db
+
+# nullarbor
+RUN wget https://github.com/tseemann/nullarbor/archive/refs/tags/v${NULLARBOR_VER}.tar.gz && \
+ tar -xzf v${NULLARBOR_VER}.tar.gz && \
+ rm v${NULLARBOR_VER}.tar.gz
+
+# set /data as working directory
+WORKDIR /data
+
+# set $PATH's
+ENV PATH="${PATH}:\
+/SPAdes-${SPADES_VER}-Linux/bin:\
+/kmc:\
+/Lighter-1.1.1:\
+/Trimmomatic-${TRIMMOMATIC_VER}:\
+/bwa-0.7.17:\
+/skesa:\
+/megahit/megahit_v1.1.4_LINUX_CPUONLY_x86_64-bin:\
+/velvet/velvet-1.2.10:\
+/flash/FLASH-1.2.11:\
+/shovill-${SHOVILL_VER}/bin:\
+/pilon:\
+/samclip:\
+/nullarbor-${NULLARBOR_VER}/bin:\
+/bedtools2/bin:\
+/barrnap-0.9/bin:\
+/prokka-${PROKKA_VER}/bin:\
+/prokka-${PROKKA_VER}/binaries/common:\
+/snippy-${SNIPPY_VER}/bin:\
+/snippy-${SNIPPY_VER}/binaries/linux:\
+/snippy-${SNIPPY_VER}/binaries/noarch:\
+/mlst-${MLST_VER}/bin:\
+/abricate-${ABRICATE_VER}/bin:\
+/mash-Linux64-v${MASH_VER}:\
+/iqtree-${IQTREE_VER}-Linux/bin/:\
+/FastTree:\
+/quicktree-${QUICKTREE_VER}:\
+/opt/kraken:\
+/kraken2-${KRAKEN2_VER}"\
+ LC_ALL=C\
+ KRAKEN_DEFAULT_DB="/kraken-db"\
+ KRAKEN2_DEFAULT_DB="/kraken2-db"\
+ CENTRIFUGE_DEFAULT_DB="/centrifuge-db"
+
+# test layer; may add more later once everything is installed
+FROM app as test 
+
+# print version and check dependencies
+RUN nullarbor.pl --version
+
+# make fake files to trick nullarbor dependency check
+RUN touch ${KRAKEN_DEFAULT_DB}/database.idx ${KRAKEN_DEFAULT_DB}/database.kdb && \
+ touch ${CENTRIFUGE_DEFAULT_DB}.1.cf ${CENTRIFUGE_DEFAULT_DB}.2.cf ${CENTRIFUGE_DEFAULT_DB}.3.cf
+
+# uncomment for final checks
+# RUN nullarbor.pl --check
```

Comment:
uncomment line 395.

Glad these parts are in the test layer, because the fake kraken db files and fake centrifuge db files will not be present in the final, deployed docker image. Forces the user to provide databases